docs/core/plugins/sieve.md

```diff
@@ -622,75 +682,92 @@ actions are simply discarded.
 For example:
 
 ```[dovecot.conf]
-plugin {
-  # Global scripts executed before the user's personal script.
-  #   E.g. handling messages marked as dangerous
-  sieve_before = /var/lib/dovecot/sieve/discard-viruses.sieve
-
-  # Domain-level scripts retrieved from LDAP
-  sieve_before2 = ldap:/etc/dovecot/sieve-ldap.conf;name=ldap-domain
-
-  # User-specific scripts executed before the user's personal script.
-  #   E.g. a vacation script managed through a non-ManageSieve GUI.
-  sieve_before3 = /var/vmail/%{user | domain}/%{user | username}/sieve-before
-
-  # User-specific scripts executed after the user's personal script.
-  # (if keep is still in effect)
-  #   E.g. user-specific default mail filing rules
-  sieve_after = /var/vmail/%{user | domain}/%{user | username}/sieve-after
-
-  # Global scripts executed after the user's personal script
-  # (if keep is still in effect)
-  #   E.g. default mail filing rules.
-  sieve_after2 = /var/lib/dovecot/sieve/after.d/
+# Global scripts executed before the user's personal script.
+#   E.g. handling messages marked as dangerous
+sieve_script before1 {
+  type = before
+  path = /var/lib/dovecot/sieve/discard-viruses.sieve
+}
+
+# Domain-level scripts retrieved from LDAP
+sieve_script before2 {
+  type = before
+  name = ldap-domain
+  driver = ldap
+  # ldap settings here: ...
+}
+
+# User-specific scripts executed before the user's personal script.
+#   E.g. a vacation script managed through a non-ManageSieve GUI.
+sieve_script before3 {
+  type = before
+  path = /var/vmail/%{user | domain}/%{user | username}/sieve-before
+}
+
+# User-specific scripts executed after the user's personal script.
+# (if keep is still in effect)
+#   E.g. user-specific default mail filing rules
+sieve_script after1 {
+  type = after
+  path = /var/vmail/%{user | domain}/%{user | username}/sieve-after
+}
+
+# Global scripts executed after the user's personal script
+# (if keep is still in effect)
+#   E.g. default mail filing rules.
+sieve_script after2 {
+  type = after
+  path = /var/lib/dovecot/sieve/after.d/
 }
 ```
 
 ::: tip
-Be sure to manually pre-compile the scripts specified by
-[[setting,sieve_before]] and [[setting,sieve_after]] by using the
-[[link,sievec]] tool.
+Be sure to manually pre-compile the scripts specified by `before` and `after`
```

Comment:
`[[link,sieve_storage_type_before,before]] and [[link,sieve_storage_type_after,after]]`

Reply:
fixed